common/persistence/elasticsearch/es_visibility_store.go

```diff
@@ -772,11 +773,11 @@ func createVisibilityMessage(
 	executionTimeUnixNano int64,
 	taskID int64,
 	memo []byte,
-	encoding common.EncodingType,
+	encoding constants.EncodingType,
```

Comment:
No action: comment only: 

I think its not great we have a type in that file, but I also understand we need it declared there in order to have constants use it, so